<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugControls</class>
 <widget class="QWidget" name="DebugControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>debug controls</string>
  </property>
  <widget class="QGroupBox" name="commands_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>command</string>
   </property>
   <widget class="QLineEdit" name="command_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="run_command_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="title">
    <string>misc</string>
   </property>
   <widget class="QCheckBox" name="show_original_checkbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>show original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>render sprite:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_zelda_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>load zelda.nes</string>
    </property>
   </widget>
   <widget class="QLabel" name="sprite_offset_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sprite pattern offset:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="render_sprite_spinbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sprite_pattern_offset_spinbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="take_shot_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>screenshot</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>command_line</sender>
   <signal>returnPressed()</signal>
   <receiver>run_command_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
